iocore/cache/CacheRead.cc

```diff
@@ -910,7 +910,7 @@ CacheVC::openReadStartEarliest(int /* event ATS_UNUSED */, Event * /* e ATS_UNUS
           f.evac_vector   = 1;
           f.use_first_key = 1;
           key             = first_key;
-          // always use od->first_dir to overwrite a directory.
+          // always use od->first_dir to overwrite a directory.e
```

Comment:
Were you trying to add "eh" to the end of the comment to make TS more Canada-friendly?